src/test/java/uk/gov/hmcts/reform/wacaseeventhandler/services/CancellationTaskHandlerTest.java

```diff
@@ -70,14 +71,16 @@ void evaluateDmn() {
 
         when(workflowApiClientToCancelTask.evaluateDmn(
             DMN_NAME,
-            requestParameters
+            requestParameters,
+            TENANT_ID
```

Comment:
What happens if tenant_id is not provided or is blank? Do we need a test to cover this?